doc/source/Quickstart.rst

```diff
@@ -71,6 +71,9 @@ The following instructions install a new spack environment on a pre-configured s
     spack concretize 2>&1 | tee log.concretize
     spack install [--verbose] [--fail-fast] 2>&1 | tee log.install
 
+.. note::
+  For platforms with multiple compilers in the site config, make sure that the correct compiler and corresponding MPI library are set ``envs/jedi-fv3.hera/site/packages.yaml`` before running ``spack concretize``. Also, check the output of ``spack concretize`` to make sure that the correct compiler is used (e.g. %intel-2022.0.1). If not, edit ``envs/jedi-fv3.hera/site/compilers.yaml`` and remove the offending compiler. Then, remove ``envs/jedi-fv3.hera/spack.lock`` and rerun ``spack concretize``.
```

Comment:
Looks like a typo in line 75. Should it be something like:
```suggestion
  For platforms with multiple compilers in the site config, make sure that the correct compiler and corresponding MPI library are set correctly in ``envs/jedi-fv3.hera/site/packages.yaml`` before running ``spack concretize``. Also, check the output of ``spack concretize`` to make sure that the correct compiler is used (e.g. %intel-2022.0.1). If not, edit ``envs/jedi-fv3.hera/site/compilers.yaml`` and remove the offending compiler. Then, remove ``envs/jedi-fv3.hera/spack.lock`` and rerun ``spack concretize``.
```

Also don't the new instructions for macOS and Linux, where you unset the SPACK_SYSTEM_CONFIG_PATH before running the `spack config add` commands, place the compiler spec in the `envs/my-env/spack.yaml` file instead of the `envs/my-env/site/package.yaml` file. Would it be worth adding a note on the Platform page about this difference? Sorry if that note is already there and I missed it.